projects/serde_urlencoded/build.sh

```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 Google LLC
+# Copyright 2023 Google LLC
```

Comment:
We usually don't change year for the existing files

Reply:
Done